lambda.py

```diff
@@ -59,6 +59,9 @@ def common_options(function):
         default=600,
         help="Timeout of the orchestrator lambda function",
     )(function)
+    function = click.option(
+        "--s3-code-bucket", default=None, help="S3 bucket where code is uploaded",
+    )(function)
```

Comment:
Ah, was this needed because the zipfile got bigger with the switch to boto3?

Reply:
Correct